cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -461,9 +536,19 @@ func (g *genConversion) generateConversion(inType, outType *types.Type, sw *gene
 	} else {
 		sw.Do(fmt.Sprintf("func auto%s(in *$.inType|raw$, out *$.outType|raw$, s conversion.Scope) error {\n", funcName), argsFromType(inType, outType))
 	}
-	sw.Do("if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {\n", nil)
-	sw.Do("defaulting.(func(*$.|raw$))(in)\n", inType)
-	sw.Do("}\n", nil)
+	if function, ok := g.defaulters[inType]; ok {
+		if function != nil {
+			sw.Do("$.|raw$(in)\n", function)
+		}
+	} else {
+		// TODO: allow to be enabled per package (generateDefaultReflection=true)
```

Comment:
To be honest, I don't think it makes sense.
I would remove the "else" block here.